src/main/features/claim/routes/check-and-send.ts

```diff
@@ -68,7 +68,7 @@ function deserializerFunction (value: any): StatementOfTruth | QualifiedStatemen
 }
 
 function getStatementOfTruthClassFor (draft: Draft<DraftClaim>): { new(): StatementOfTruth | QualifiedStatementOfTruth } {
-  if (draft.document.claimant.partyDetails.isBusiness() || draft.document.claimant.partyDetails.isSoleTrader()) {
+  if (draft.document.claimant.partyDetails.isBusiness()) {
```

Comment:
Shall we not extend route test e.g. `it('should render page when everything is fine', async () =>` so that we statement or truth text is checked for different cases?